tests/entrypoint.sh

```diff
@@ -23,4 +23,6 @@ if [ -n "$TENANT_TOKEN" ]; then
 fi
 
 /etc/init.d/ssh start
-mender --no-syslog daemon
+mender-auth daemon &
+sleep 10
+mender-update daemon
```

Comment:
Why 10 seconds, isn't it more like 1 (if even that)?

Reply:
Completely out of the blue, I haven't even run the container... :sweat_smile:

As this still depends on D-Bus being finalized I'll set it to 1 and make a mental note to test it a bit later (we use it in some tests, so we should catch errors here anyway).